test/helper-async.js

```diff
@@ -82,7 +82,7 @@ function pm(args, env, callback) {
     console.log('pmcli:', pmcli, args);
     cleanup = once(cleanup);
 
-    pm = cp.spawn(pmcli, args, {
+    pm = cp.fork(pmcli, args, {
```

Comment:
@sam-github I have some vague recollection of proposing a similar change (spawn to fork) and there being a reason against it.. what is here? does that sound familiar at all?